gpu/Dockerfile

```diff
@@ -242,12 +242,12 @@ RUN echo "===> Install ESPnet dependencies" && \
 
 WORKDIR /
 
-# Basic ESPnet installation (for CUDA and installers stuff).
-RUN git clone --depth=1 https://github.com/espnet/espnet
+RUN git clone --depth=1 -b elpis https://github.com/persephone-tools/espnet.git
```

Comment:
There should probably be a comment here explaining why it was necessary to clone the separate branch and pull bits and pieces of it.